src/libstd/alloc.rs

```diff
@@ -279,7 +279,7 @@ pub fn rust_oom(layout: Layout) -> ! {
     let hook: fn(Layout) =
         if hook.is_null() { default_alloc_error_hook } else { unsafe { mem::transmute(hook) } };
     hook(layout);
-    unsafe { crate::sys::abort_internal() }
+    crate::sys::abort_internal()
```

Comment:
Hm, it probably makes sense to call `std::process::abort()` here? Not sure why we'd want to reach into sys unnecessarily.

Reply:
Done.